stable/snap/snapcraft.yaml

```diff
@@ -63,7 +63,14 @@ parts:
         echo $version > $file
       fi
       echo "getting powershell $version"
-      curl -L -o powershell.tar.gz https://github.com/PowerShell/PowerShell/releases/download/v$version/powershell-$version-linux-x64.tar.gz
+      case "$SNAPCRAFT_ARCH_TRIPLET" in
```

Comment:
Perhaps we only make these changes to preview at first to minimize risk.

Reply:
ok, I'll drop the stable snapcraft for the time being